pyxis/find_signatures.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+"""
+Python script to find all signatures in Pyxis for a given repository and manifest_digest
+and save a list of references that are signed in a text file, one reference per line
+
+Required env vars:
+PYXIS_KEY_PATH
+PYXIS_CERT_PATH
+
+Optional env vars:
+PYXIS_GRAPHQL_API
+"""
+import argparse
+import logging
+import os
+
+import pyxis
+
+LOGGER = logging.getLogger("find_signatures")
+
+
+def parse_arguments() -> argparse.Namespace:  # pragma: no cover
+    """Parse CLI arguments
+
+    :return: Dictionary of parsed arguments
+    """
+
+    parser = argparse.ArgumentParser(
+        description="find all signatures in Pyxis for a given repository and manifest_digest"
+                    "and save a list of references that are signed in a text file, one reference"
+                    "per line"
```

Comment:
Black failed on the indentation here, in case you haven't noticed.
```suggestion
        "and save a list of references that are signed in a text file, one reference"
        "per line"
```
(I hope I removed the right amount of spaces :) )